master: optimizer: update default value of fix control 52869 (#23141)
pingcap/docs: optimizer-fix-controls.md

### [`52869`](https://github.com/pingcap/tidb/issues/52869) <span class="version-mark">New in v8.1.0</span>

> - Default value: `ON`. Before v8.5.7, the default value is `OFF`.

Reviewer: ```suggestion
- Default value: `ON`. Before v8.5.7 and v9.0.0, the default value is `OFF`.
```

- Possible values: `ON`, `OFF`

> - When this fix control is set to `OFF`, if the optimizer can choose the single index scan method (other than full table scan) for a query plan, the optimizer will not automatically use index merge. For more details, see the **Note** in [Explain Statements Using Index Merge](/explain-index-merge.md#examples).

Reviewer: ```suggestion
- When this fix control is set to `OFF`, if the optimizer can choose the single index scan method (other than full table scan) for a query plan, the optimizer does not automatically choose index merge. For more information, see the **Note** in [Explain Statements Using Index Merge](/explain-index-merge.md#examples).
```
